modules/restricted-guests/synapse/Dockerfile

```diff
@@ -1,7 +1,12 @@
 ARG DEBIAN_VERSION_NUMERIC=12
+
+# Now copy it into our base image.
+FROM busybox AS build
```

Comment:
While I think this is fine given it will be statically linked, I wonder if it would be better to use `gcr.io/distroless/base-nossl-debian${DEBIAN_VERSION_NUMERIC}:debug` and then copy across `/busybox/cp` to `/bin/cp`